rapids/internal/launch-dask.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# This script creates systemd configuration for Dask.
+
+set -euxo pipefail
+
+echo "Installing Dask service..."
+
+readonly ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
+readonly MASTER=$(/usr/share/google/get_metadata_value attributes/dataproc-master)
+readonly RUN_CUDA_WORKER_ON_MASTER=$(/usr/share/google/get_metadata_value attributes/run-cuda-worker-on-master || echo -n 'true')
+readonly DASK_LAUNCHER='/usr/local/bin/launch-dask.sh'
+readonly INIT_SCRIPT='/usr/lib/systemd/system/dask-cluster.service'
+# For Anaconda Dataproc component
+readonly PREFIX='/opt/conda/anaconda/envs/RAPIDS/bin'
+# For miniconda dataproc init-action
+#readonly PREFIX='/opt/conda/bin'
```

Comment:
I would suggest to support this with a metadata flag or remove these comments. Not sure if those can be helpful for someone trying to use conda, jupyter init-actions instead. (more such places).

Reply:
Comments removed